grails-app/views/organisation/_organisationReportHeader.gsp

```diff
@@ -16,11 +16,15 @@
 
     <div class="row mb-2">
         <div class="col-sm-2 header-label">Reporting period end</div>
-
-        <div class="col-sm-9 value"><g:formatDate format="dd MMM yyyy"
-                                         date="${au.org.ala.merit.DateUtils.parse(report.toDate).toDate()}"/></div>
+        <g:if test="${report.toDate < context.config.organisationReports.periodEnd[0]}">
```

Comment:
This doesn't look right to me - periodEnd isn't an array so this will evaluate to null.

This value should be derived from the reporting tab "periodEnd" however it looks like we might have already taken this into account for organisations (the issue I was discussing with you was for projects rather than organisations). See https://github.com/AtlasOfLivingAustralia/fieldcapture/blob/dev/grails-app/assets/javascripts/organisation.js#L354

I feel this change was working "by accident" - because endDate is not an array, it evaluates to null so the condition is never true.  It should be deleted.

Reply:
I was wrong about this - the reason it works is due to groovy gpath/auto-spread of the organisationReports array.

Reply:
I've added a comment to clarify how/why it works as it does.